configuration/eslint-config/package.json

```diff
@@ -25,5 +23,8 @@
   },
   "files": [
     "index.js"
-  ]
+  ],
+  "devDependencies": {
+    "eslint-import-resolver-typescript": "^3.5.3"
```

Comment:
why this package is a dev dependency?

Reply:
needed for support for typescript import rules: https://www.npmjs.com/package/eslint-import-resolver-typescript